exchange/utils_test.go

```diff
@@ -3476,3 +3478,306 @@ func TestCleanOpenRTBRequestsFilterBidderRequestExt(t *testing.T) {
 		}
 	}
 }
+
+type GPPMockSection struct {
+	sectionID constants.SectionID
+	value     string
+}
+
+func (gs GPPMockSection) GetID() constants.SectionID {
+	return gs.sectionID
+}
+
+func (gs GPPMockSection) GetValue() string {
+	return gs.value
+}
+
+func TestGdprFromGPP(t *testing.T) {
+	testCases := []struct {
+		name            string
+		desc            string
+		initialRequest  *openrtb2.BidRequest
+		gpp             gpplib.GppContainer
+		expectedRequest *openrtb2.BidRequest
+	}{
+		{
+			name:            "Empty",
+			desc:            "Empty Request",
```

Comment:
Consider removing the desc field and use a comment beside the name instead:

```
name: "empty" // empty request
```

Reply:
That is an idea. But is it worth the work to move all the descriptions to comments in a test? It doesn't change the performance of the final program.

Reply:
It doesn't impact performance. It's a little odd IMHO to see a `desc` field with no usage. We used to use it in generating assert messages, but that's no longer necessary with using `t.Run`.